test/powershell/engine/Basic/DefaultCommands.Tests.ps1

```diff
@@ -173,10 +173,10 @@ Describe "Verify approved aliases list" -Tags "CI" {
 "Alias",        "write",                            "Write-Output",                     $($FullCLR -or $CoreWindows              ),     "ReadOnly",             "",                     ""
 "Cmdlet",       "Add-Computer",                     "",                                 $($FullCLR                               ),     "",                     "",                     ""
 "Cmdlet",       "Add-Content",                      "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "Medium"
-"Cmdlet",       "Add-History",                      "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "Medium"
-"Cmdlet",       "Add-Member",                       "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "Medium"
+"Cmdlet",       "Add-History",                      "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
```

Comment:
Could you please clarify why does the change need?

Reply:
This change reflects the cmdlets actual defined attributes. 

Prior to this PR they would report as having "Medium" impact when querying the attribute's set ConfirmImpact as that is the default value, despite not implementing ShouldProcess in the first place nor declaring support for it.

This is primarily a change in how this value reports when `SupportsShouldProcess` is not set to `true` for a given command, in the interest of visibility for the actual effective ConfirmImpact level of a cmdlet.

All changes from `Medium` to `None` are reflective of this. The only cmdlets that had an actual effective ConfirmImpact change

Reply:
Thanks! Please add this to the PR description.